recipe/meta.yaml

```diff
@@ -1,32 +1,35 @@
-{% set version = "0.7.3" %}
+{% set version = "0.8.0" %}
 
 package:
   name: hvplot
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/h/hvplot/hvplot-{{ version }}.tar.gz
-  sha256: 74b269c6e118dd6f7d2a4039e91f16a193638f4119b4358dc6dbd58a2e71e432
+  sha256: 8630dba34969b105e267cbe14237e2d56d9969c801980ae5e5190bc18b2792d6
 
 build:
-  number: 1
-  noarch: python
+  number: 0
+  # ModuleNotFoundError: No module named 'hvplot.__main__'
+  # entry_points:
+  #   - hvplot = hvplot.__main__:main
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python
+    - pyct
     - pip
-    - wheel
     - setuptools
-    - pyct >=0.4.4
-    - param >=1.6.1
+    - wheel
   run:
     - python
-    - bokeh >=1.0.0
+    - packaging
+    - param >=1.12.0
+    - bokeh >=2.0.0
     - colorcet >=2
-    - holoviews >=1.11.0
-    - numpy >=1.15
+    - holoviews >=1.12.0
+    - numpy
```

Comment:
`numpy>=1.15`
https://github.com/holoviz/hvplot/blob/v0.8.0/setup.py#L35

Reply:
@skupr-anaconda  Will pinning `numpy` in the `meta.yaml` cause problems with the `cbc.yaml`?

Reply:
It would cause problems if `numpy` would be in `host`. Because `cbc.yaml` is a matrix of packages for `host`. But in `run` we should specify the minimum version of numpy to be used `>=1.15`. Generally, it will be used the latest version in `run`.

Reply:
Done, thanks!